packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -13,17 +13,22 @@ export interface UpdateVersionInfo {
   updateAvailable: boolean;
 }
 
-export default interface Environment {
+export default interface BackendEnvironment extends FrontEndEnvironment {
   cwd: string;
+  prismicData: PrismicData;
+  client: DefaultClient | FakeClient;
+}
+
+export interface FrontEndEnvironment {
   userConfig: UserConfig;
-  hasConfigFile: boolean;
   repo?: string;
-  prismicData: PrismicData;
   chromatic?: Chromatic;
   updateVersionInfo: UpdateVersionInfo;
   mockConfig: any;
   framework: Framework;
```

Comment:
only used in "LackStoryBook" component, to consider removing after the preview stuff :D

Reply:
👍